src/Lightbox.js

```diff
@@ -252,7 +267,7 @@ class Lightbox extends Component {
 
 		const thumbnailsSize = showThumbnails ? this.theme.thumbnail.size : 0;
 		const heightOffset = `${this.theme.header.height + this.theme.footer.height + thumbnailsSize
-			+ (this.theme.container.gutter.vertical)}px`;
+		+ (this.theme.container.gutter.vertical)}px`;
```

Comment:
Is lint happy with this ?

Reply:
Yes lint is happy. If you doubt me, checkout my repo and lint it yourself.